### PR TITLE
Documentation: Fix typos and improve docs on report `status.conditions`

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

File: Documentation/report.md [hunk 1]
```diff
@@ -284,7 +284,7 @@ The execution of a scheduled report can be tracked using its status field. Any e
 
 The `status` field of a `Report` currently has two fields:
 
-- `conditions`: Conditions is an list of conditions, each have a `Type`, `Reason`, and `Message` field. Possible values of a condition's `Type` field are `Running` and `Failure`, indicating the current state of the scheduled report. The `Reason` indicates why its `Condition` is in its current state, with and the `Message` provides a detailed information on the `Reason`.
+- `conditions`: Conditions is a list of conditions, each have a `type`, `status`, `reason`, and `message` field. Possible values of a condition's `type` field are `Running` and `Failure`, indicating the current state of the scheduled report. The `reason` indicates why its `condition` is in its current state with it's `status` being either `true`, `false` or `unknown`. The `message` provides a human readable indicating why the condition is in the current state. For detailed information on the `reason` values see https://github.com/operator-framework/operator-metering/blob/master/pkg/apis/metering/v1alpha1/util/report_util.go#L10.
```
Comment (tschuy) [inline Documentation/report.md]: should make this a pretty link, maybe "see `v1alpha1/util/report_util.go`"

Comment (tschuy) [inline Documentation/report.md]: minor grammar fix, 
"Conditions is a list of conditions, each have a `type`, `status`, `reason`, and `message` field" ->
"Conditions is a list of conditions; each have a `type`, `status`, `reason`, and `message` field" or
"Conditions is a list of conditions, each of which have a `type`, `status`, `reason`, and `message` field"

Comment (chancez) [reply Documentation/report.md]: Fixed

Comment (chancez) [reply Documentation/report.md]: Went with number 3

**Pull request merged**